spec/models/report_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe Report, type: :model do
+  let!(:user) { create(:user) }
+
+  describe '#editable?' do
+    let!(:report) { create(:report, user: user) }
+
+    context 'when the user is the owner of the report' do
+      it 'returns the true' do
```

Comment:
```suggestion
      it 'is editable' do
```

みたいな書き方の方が抽象度が一段高くて良いかもしれません。
https://bootcamp.fjord.jp/articles/78

Reply:
対しかにtrueを返すだと、まんまコードのままですね。
変更しました。